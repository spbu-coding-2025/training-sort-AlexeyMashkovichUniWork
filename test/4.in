-839152734 -169136779 -917039722 687915114 -358781099 536495264 -133710517 837000086 -682228933 393369546 -934907442 -44806871 632409832 82044361
