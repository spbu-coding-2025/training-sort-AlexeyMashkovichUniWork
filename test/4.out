-934907442 -917039722 -839152734 -682228933 -358781099 -169136779 -133710517 -44806871 82044361 393369546 536495264 632409832 687915114 837000086
